static/app/utils/formatters.tsx

```diff
@@ -289,16 +289,25 @@ type ParsedLargestSuffix = [val: number, suffix: moment.unitOfTime.DurationConst
  */
 export function parseLargestSuffix(
   seconds: number,
-  maxSuffix: string = 'days'
+  maxSuffix: string = 'days',
+  abbreviation: boolean = false
```

Comment:
what if there were some other function to convert from the units/levels into the abbrs?

then this wouldn't get wider, and other things could compose the little mapping.